modules/swagger-codegen/src/main/java/io/swagger/codegen/InlineModelResolver.java

```diff
@@ -191,6 +191,16 @@ public void flatten(Swagger swagger) {
         }
     }
 
+    private void fixStringModel(ModelImpl m) {
+        if (m.getType() != null && m.getType().equals("string") && m.getExample() != null) {
+            String example = m.getExample().toString();
+            if (example.substring(0, 2).equals("\"") &&
+                    example.substring(example.length() - 2).equals("\"")) {
+                m.setExample(example.substring(2, example.length() - 2));
```

Comment:
You are right... I **really** don't understand how it worked for me (tested it a couple of times...) but now, it doesn't work.
I will submit your fix in a couple of minutes..